client/src/pages/innerPages/myCreate/index.js

```diff
@@ -17,16 +18,49 @@ export default function MyCreate() {
         parent: ""
     })
 
-  
+    //removes a tag or skill from the appropriate State
+    function set(arrType, cut) {
+        if (arrType === tags) {
+            setTags(arrType => arrType.filter((arrType, i) => i !== cut))
+        }
+        else if (arrType === skills) {
+            setSkills(arrType => arrType.filter((arrType, i) => i !== cut));
+        }
+        else {
+            console.log(arrType)
+        }
+    }
+
+    //takes the array and a string representation of the array 
+    //makes out the tags and skills on page
+    function mapping(arrType, string) {
```

Comment:
Could you comment on the name of this function "mapping"? Looks different than the other function names so just wanted to make sure it's intentional.

Reply:
Yes I will in the next push, I struggled with naming this as it is mapping the tags and the skills. I suppose I could do something like mapTagsOrSkills but it could take any array and map it's contents so I'm not sure...